app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] Great job so far :clap:  But I noticed that the comments controller doesn't have a new action. :cry:  Please make sure that the comments controller has new and create actions similar to the post controller in order to make your app work as expected :+1: